docs/en/user_guides/deploy.md

```diff
@@ -1,4 +1,212 @@
 # Model Deployment
 
-MMDet 3.x fully relies on [MMDeploy](https://mmdeploy.readthedocs.io/) to deploy models.
-Please stay tuned and this document will be update soon.
+- [Installation](#installation)
+  - [Install mmdet](#install-mmdet)
+  - [Install mmdeploy](#install-mmdeploy)
+- [Convert model](#convert-model)
+- [Model specification](#model-specification)
+- [Model inference](#model-inference)
+  - [Backend model inference](#backend-model-inference)
+  - [SDK model inference](#sdk-model-inference)
+- [Supported models](#supported-models)
+
+______________________________________________________________________
+
+The deployment of OpenMMLab codebases, including MMDetection, MMClassification and so on are supported by [MMDeploy](https://github.com/open-mmlab/mmdeploy),
+who provides the fullest and latest deployment guides that can be found from [here](https://mmdeploy.readthedocs.io/en/dev-1.x/).
```

Comment:
Would it be better if we directly refer to this link: https://mmdeploy.readthedocs.io/en/dev-1.x/04-supported-codebases/mmdet.html